datascience/tables.py

```diff
@@ -5678,12 +5678,46 @@ def asdict(self):
             return collections.OrderedDict(zip(self._table.labels, self))
 
     class Rows(collections.abc.Sequence):
-        """An iterable view over the rows in a table."""
+        """
+        An iterable view over the rows in a table.
+
+        args:
+        table- accepts a table object of class Table()
+        
+        >>> from datascience import Table
+        >>> t = Table().with_columns({
+             'letter': ['a', 'b', 'c', 'z'],
+             'count':  [  9,   3,   3,   1],
+             'points': [  1,   2,   2,  10],
+        })
+
+        >>> r = Table.Rows(t)
+            print(r)
```

Comment:
In doctests, we don't need to `print` any values. You can simply call `>>> r` and it will output the right representation of the object.